webapp/app/css/_header.less

```diff
@@ -20,4 +20,12 @@ header {
       width: 24px;
     }
   }
-}
\ No newline at end of file
+
+  .header1 {
+    margin: 0 0 0 0;
+  }
+
+  .header2 {
+    margin: 0 0 16px 0;
+  }
```

Comment:
I'd just add padding to the bottom of the `header` tag and use divs (without any default styles) to create breaks:

```
<div>IKE, Getting Started</div>
<div>Learn more...</div>
```

```
header {
   padding-bottom: 16px;
}
```